disperser/dataapi/nonsigner_handler.go

```diff
@@ -85,13 +91,29 @@ func (s *server) getOperatorNonsigningRate(ctx context.Context, intervalSeconds
 				if err != nil {
 					return nil, err
 				}
+
+				opID := [32]byte{}
+				opIDslice, err := hex.DecodeString(op[2:])
+				if err != nil {
+					return nil, err
+				}
+				copy(opID[:], opIDslice)
+
+				const multipler = 10000
+				stakePercentage := float64(0)
+				if stake, ok := state.Operators[q][opID]; ok {
+					p, _ := new(big.Int).Div(new(big.Int).Mul(stake.Stake, big.NewInt(multipler)), state.Totals[q].Stake).Float64()
+					stakePercentage = p / multipler
+				}
+
 				nonsignerMetric := OperatorNonsigningPercentageMetrics{
 					OperatorId:           fmt.Sprintf("0x%s", op),
 					OperatorAddress:      nonsignerIdToAddress[op],
 					QuorumId:             q,
 					TotalUnsignedBatches: unsignedCount,
 					TotalBatches:         totalCount,
 					Percentage:           pf,
+					StakePercentage:      stakePercentage,
```

Comment:
If this field is 0, it indicates the operator is no longer in the active set at block endBlock?

Reply:
Yes, that's the correct interpretation.